Zend/zend_compile.c

```diff
@@ -3956,19 +3956,21 @@ static bool zend_compile_call_common(znode *result, zend_ast *args_ast, zend_fun
 	if (args_ast->kind == ZEND_AST_CALLABLE_CONVERT) {
 		opline = &CG(active_op_array)->opcodes[opnum_init];
 		opline->extended_value = 0;
+		/* opcode array may be reallocated, so don't access opcode field after zend_emit_op_tmp(). */
+		uint8_t opcode = opline->opcode;
 
-		if (opline->opcode == ZEND_NEW) {
+		if (opcode == ZEND_NEW) {
 		    zend_error_noreturn(E_COMPILE_ERROR, "Cannot create Closure for new expression");
```

Comment:
non-PR touched code has wrong indent this line

Reply:
Huh indeed. Good find. I'll fix it separately.

Reply:
Fixed via https://github.com/php/php-src/commit/f9c4fe4c5d337e92fd0a6df506385dc645b7394a